mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/149 - zone property panel (Merge this second!)

Closes #149 

To test:
- Run introductory scenario
- Click on Zone View and select Zone
- Make sure properties appear correctly (except Security Rating)

Notes:
- Calculating the security rating is very complicated, so I'm making it its own case
